pkg/network/node/ovscontroller.go

```diff
@@ -923,3 +926,17 @@ func (oc *ovsController) SetNamespaceEgressViaEgressIPs(vnid uint32, egressIPsMe
 	}
 	return otx.Commit()
 }
+
+func ensureLocalMAC(localIP string) string {
+	interfaces, _ := net.Interfaces()
+	for _, interf := range interfaces {
+		if addrs, err := interf.Addrs(); err == nil {
+			for _, addr := range addrs {
+				if strings.Contains(addr.String(), localIP) {
+					return interf.HardwareAddr.String()
+				}
+			}
+		}
+	}
+	return ""
```

Comment:
Should this be an error instead? Or do you expect a scenario in which `localMAC` is empty?

Reply:
I add the check at line 272. PTAL